requirements-dev.txt

```diff
@@ -18,3 +18,4 @@ six==1.12.0               # via cfgv, pip-tools, pre-commit
 toml==0.10.0              # via pre-commit
 virtualenv==16.7.5        # via pre-commit
 zipp==0.6.0               # via importlib-metadata
+coverage==4.5.4           # need it for running tests
```

Comment:
This file is autogenerated and shouldn't be modified manually. Create separate file for test requirements instead (`tests/requirements.txt`)